package.json

```diff
@@ -49,7 +49,15 @@
       "ecmaVersion": 13,
       "sourceType": "module"
     },
-    "rules": {}
+    "rules": {
+      "no-unused-vars": [
+        "error",
+        {
+          "varsIgnorePattern": "^_",
+          "argsIgnorePattern": "^_"
```

Comment:
I use `varsIgnorePattern` in this PR, but not `argsIgnorePattern`. I added it for consistency.